distribution/openrct2.d.ts

```diff
@@ -2674,6 +2674,13 @@ declare global {
         launch(launchData?: CrashParticleLaunchData): void;
     }
 
+    enum PeepDirection {
+        north = 0,
+        east = 1,
+        south = 2,
+        west = 3
+    };
```

Comment:
IIRC this would need to be implemented on the C++ side as well (see TrackSlope and TrackBanking).

Also enum members on the TS side are PascalCase. 🙂

Reply:
> Any particular reason? In the end, this is just downcast/transpiled to an integer, isn't it?

I think it's because it's only transpiled if it is in an actual `ts` file. In a `d.ts` file it's not because normally a `d.ts` file is expected to be backed by a `js` file with the implementation. A 

A `d.ts` is just a declaration of what non-TS implementations are already available and is not included/re-transpiled again.

Reply:
Discussed this with @Basssiiie on Discord and decided to go with the existing `Direction` type instead of creating a new one.